Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -80,6 +80,11 @@ public bool IsActiveInstance
         /// <summary>
         /// The public property of the Active Profile, ensuring events are raised on the change of the configuration
         /// </summary>
+        /// <remarks>
+        /// When setting the ActiveProfile, all currently running services will be destroyed and those specified in
+        /// the new profile will be instantiated and initialized. A noticable application hesitation may occur
+        /// during this process.
```

Comment:
There's going to be time between LateUpdate and Update where MRTK is destroyed across calls to update the ActiveProfile - it woudl be good to capture that in the remarks.

Also would be good to mention that any scripts that have higher priority than the MRTK object will run before this has a chance to set things, so to be careful with accessing MRTK state if you happen to have such a thing.